astropy/io/fits/convenience.py

```diff
@@ -709,6 +710,111 @@ def info(filename, output=None, **kwargs):
     return ret
 
 
+def printdiff(inputa, inputb, *args, **kwargs):
+    """
+    Compare two parts of a FITS file, including entire FITS files,
+    FITS `HDUList` objects and FITS ``HDU`` objects.
+
+    Parameters
+    ----------
+    inputa : str, `HDUList` object, or ``HDU`` object
```

Comment:
Does
```
`HDUList`
```
really link to its API properly? Or was it intended to be double-backticks?